### DISCUSSION TITLE
next/image: internal server error with signed url

### DISCUSSION THREAD

**Discussion by emulienfou** [vercel/next.js#19449] [Help]
Hello there,
having an issue when trying to use the new `next/image` with external url from Google Cloud Storage.

My image url is a Google Cloud Signed Url, however next return a 500 internal server error when deployed on Vercel infrastructure but works find in my local.

**Here is the next/image generated url:** https://next-js-git-dev.edmdb.vercel.app/_next/image?url=https%3A%2F%2Fstorage.googleapis.com%2Fedm-passport%2Fuploads_dev%2Fpodcast%2F1ef8f875aa3731e38c32be7075e5357e%2540112x112.jpeg%3FGoogleAccessId%3Dsymfony%2540edm-passport.iam.gserviceaccount.com%26Expires%3D1606237234%26Signature%3DkBHMtbOiaDcd%252BoD%252Fkxm57PvCIPx8oqhqi%252B73phgHrpw6CIgKfzywSDDOWhbArjH6P8x8WSOadJtah%252BKlFXpXEmRj%252FAsw%252BqsNi9jbchyOs7d0L0lsZiDTtAmgXlxzh4L5BFXbdgnBMrSjk7OicHTwxsyxBvTAjuzctkRVAsC8Ti3xsaohzk9rjBFoBmoEzgP5IHhcuU3qBxWyI9Yatsap%252B1U8I9M9M52y8fBawaDPqDEmIjdp%252FHpWGiQpYag%252BZPTALZKLGkszdFdod67LO6Zhi428LTKJBBCbuq15m0ZeRsh%252F0hzoXBMgS1uUlYGzzMhrSlyEUvQIVx4Th%252FT73qIDJw%253D%253D&w=128&q=75

**Here is the image url available on my Google Cloud Storage:** https://storage.googleapis.com/edm-passport/uploads_dev/podcast/1ef8f875aa3731e38c32be7075e5357e%40112x112.jpeg?GoogleAccessId=symfony%40edm-passport.iam.gserviceaccount.com&Expires=1606237234&Signature=kBHMtbOiaDcd%2BoD%2Fkxm57PvCIPx8oqhqi%2B73phgHrpw6CIgKfzywSDDOWhbArjH6P8x8WSOadJtah%2BKlFXpXEmRj%2FAsw%2BqsNi9jbchyOs7d0L0lsZiDTtAmgXlxzh4L5BFXbdgnBMrSjk7OicHTwxsyxBvTAjuzctkRVAsC8Ti3xsaohzk9rjBFoBmoEzgP5IHhcuU3qBxWyI9Yatsap%2B1U8I9M9M52y8fBawaDPqDEmIjdp%2FHpWGiQpYag%2BZPTALZKLGkszdFdod67LO6Zhi428LTKJBBCbuq15m0ZeRsh%2F0hzoXBMgS1uUlYGzzMhrSlyEUvQIVx4Th%2FT73qIDJw%3D%3D

How can I debug thing like this?
Do you think signed url are not supported by next/image?

Thanks

**Comment by binajmen** [comment]
Hi @emulienfou,

Having the same issue. Did you find a solution to your problem?

**Reply by emulienfou** [reply]
Hi @binajmen sorry I did not find a solution last time for this issue.

Switched from Google to AWS and don't host my project on Vercel anymore.

Using AWS S3 bucket now, everything works fine, no issues anymore!

**Reply by binajmen** [reply]
Mmmh okay, I sincerely think about switching to AWS also, as it seems hosting Next JS on Lambda is possible, where hosting on Cloud Functions is not (at least, in europe region).

I just like how GCP manage all the pieces together. Will have to learn a bit more about AWS first.

In the meantime, I'll use the good old `<img>` then...

Something totally unrelated, but if you have good resources/tips you based yourself on to switch from GCP to AWS, please do share them ;)

Thanks for the quick reply
